src/frontend/components/Header/Header.js

```diff
@@ -13,15 +13,14 @@ export default function Header() {
             <img className="logo-img" src={logo} alt="logo" />
           </Link>
         </div>
-        <div className="navbar-div nonDisplayonProfile">
-          <span className="sign-up" >
+        <div className="join-container">
```

Comment:
Please see my comment for this name convention regarding the className

Reply:
changed them